lib/events/test/suite.go

```diff
@@ -335,9 +354,15 @@ func (s *EventsSuite) SearchSessionEvensBySessionID(t *testing.T) {
 		require.Equal(t, e.GetSessionID(), secondID)
 	}()
 
+	queryTimeout := s.SearchSessionEvensBySessionIDTimeout
+	if queryTimeout == 0 {
+		queryTimeout = time.Second * 10
+	}
+
+	// TODO(tobiaszheller): drop it when ctx is propagated to search calls.
```

Comment:
https://github.com/gravitational/teleport/blob/48997836440987ecb0b44d142bdfd620dd290020/lib/events/test/suite.go#L346-L367
this whole part is done to provide timeout in tests (it was done before). When we propagate ctx to search calls, it can be simplified by providing ctx with timeout.